core/src/components/chip/chip.tsx

```diff
@@ -36,14 +36,22 @@ export class Chip implements ComponentInterface {
    */
   @Prop() disabled = false;
 
+  /**
+   * Define the Chip corner shape, when using the Ionic Theme.
+   */
+  @Prop({ reflect: true }) shape?: 'round' | 'rectangular';
```

Comment:
```suggestion
  @Prop() shape?: 'round' | 'rectangular';
```

The `reflect` option will set the attribute on the HTML element. Often used for styling concerns or if you have JS logic that queries based on an attribute. 

Since we are using `.chip-round`, etc. for styling, we don't need to reflect here from what I can tell.